charts/t8s-cluster/templates/workload-cluster/_helmRelease.yaml

```diff
@@ -1,6 +1,5 @@
 {{- define "t8s-cluster.helm.resourceIntoCluster" -}}
   {{- $_ := mustMerge . (pick .context "Values" "Release" "Chart") -}}
-apiVersion: helm.toolkit.fluxcd.io/v2beta2
```

Comment:
Why did you delete this?

Reply:
Oh. Not on purpose.

Guess this was a typo in vim.

Reply:
Fixed it.